recipes-kernel/linux/linux-nezha-dev.bb

```diff
@@ -0,0 +1,22 @@
+require linux-mainline-common.inc
+FILESEXTRAPATHS:prepend = "${FILE_DIRNAME}/linux-nezha:"
+SUMMARY = "Nezha dev kernel recipe"
+
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
```

Comment:
With https://github.com/riscv/meta-riscv/pull/324 you shouldn't need this any more